node-fs-writefile/cp.js

```diff
@@ -0,0 +1,8 @@
+import { writeFile, readFile } from 'node:fs/promises';
+try {
+  const [, , ...fileList] = process.argv;
```

Comment:
It would make the code more clear to explicitly name the two parameters rather than putting them in a list and accessing them by index. Something like `const [, , src, dst] = ...`